src/js/server/utils.js

```diff
@@ -17,6 +17,15 @@ var request = require("request"); // TODO:  Replace this with a writable data so
 var gpii = fluid.registerNamespace("gpii");
 fluid.registerNamespace("gpii.express.user.utils");
 
+/**
```

Comment:
I tend to be a bit nervous using JSDocs-style block comments for things other than functions, as someone can easily trip a linting error by adding a function.  However, I wanted to ask about your docs creation work and how you would make use of isolated blocks like this there.